### ISSUE TITLE
add ability to import non standard naws format fields to imports

### ISSUE THREAD

**Issue by pjaudiomv** [bmlt-enabled/bmlt-root-server#258]
It would be nice to be able to import a csv of meetings of any bmlt field. for instance if i add duration_time column to my csv. I realize this could be done in many ways. maybe first would be limited to certain fields that would not conflict with a naws format and just be added columns to a traditional naws format

**Comment by ChrisMarshallNY** [comment]
I agree, but getting a reasonable UX to express that might be a challenge, especially if we would have the ability to choose multiple fields.

**Comment by pjaudiomv** [comment]
This would only be for importing a csv, you would just add the additional field you wanted to import as an additional column. The column name would be identical to a bmlt field name. So no ux changes needed. Except maybe error handling notices

**Comment by pjaudiomv** [comment]
It would operate on existing naws import functionality and ux not a separate ux for separate  import

**Comment by radius314** [comment]
The simplest approach would be just to import them if they match and existing field name, and if they don’t match exactly, error with the reason.
